src/ecdsa_impl.h

```diff
@@ -66,7 +66,7 @@ static int secp256k1_der_read_len(const unsigned char **sigp, const unsigned cha
         return -1;
     }
     /* X.690-207 8.1.3.5 long form length octets */
-    lenleft = b1 & 0x7F;
+    lenleft = b1 & 0x7F; /* lenleft is at least 1 */
```

Comment:
ACK, `b1` comes from an unsigned char, and the function returns before this point if `b1` < 0x80 (ruling out a value of 0) or `b1` == 0x80 (the only value that would yield a 0 from this mask)